README.md

```diff
@@ -21,7 +21,7 @@ recommend using the [app skeleton](https://github.com/cakephp/app) as
 a starting point. For existing applications you can run the following:
 
 ``` bash
-$ composer require cakephp/cakephp:"~3.2"
+$ composer require cakephp/cakephp:"~3.3"
```

Comment:
Tilde is kind of nice as it will auto update to 3.4 which I don't think caret does.